cmd/pets/cmd.go

```diff
@@ -22,12 +23,47 @@ func init() {
 	initUpCmd()
 }
 
+func Execute() error {
+	var analyticsCmd *cobra.Command
+	var err error
+
+	analyticsService, analyticsCmd, err = analytics.Init(petsAppName)
+	if err != nil {
+		return err
+	}
+
+	status, err := analytics.OptStatus()
+	if err != nil {
+		return err
+	}
+
+	if status == analytics.OptDefault {
+		fmt.Fprintf(os.Stderr, "Send anonymized usage data to Windmill (y)? ")
```

Comment:
i'm not used to seeing the prompt like this, might be clearer to do: `[Y/n]`

Reply:
done!